### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by skimo-openhub**

### CODE REVIEW

File: cuda.c [hunk 1]
```diff
@@ -412,7 +412,7 @@ static __isl_give isl_printer *print_type_strip_const(__isl_take isl_printer *p,
 		size_t length = full_length - sizeof(kw_const) + 1;
 		size_t prefix_length = pos - var->array->type;
 		type = isl_alloc_array(isl_printer_get_ctx(p),
-			char, length);
+			char, length + 1);
```
Comment (ftynse) [inline cuda.c]: Isn't `+1` in `size_t length = full_length - sizeof(kw_const) + 1;`  (line 412) enough?

Comment (skimo-openhub) [reply cuda.c]: It's certainly not enough for the

```
type[length] = 0;
```

but perhaps that one should be fixed then.

Comment (skimo-openhub) [reply cuda.c]: Having looked at it more closely, the "+1" only compensates for the '\0' in sizeof(kw_const).
That is, there's no extra element subtracted, but there's also not extra room created.

Comment (ftynse) [reply cuda.c]: 👍

**Pull request merged**